docker/k8s/vtadmin/nginx.conf

```diff
@@ -0,0 +1,43 @@
+events {}
+http {
+    include mime.types;
```

Comment:
add `server_tokens off; ` to remove sensitive information about the nginx version.